operators/index.js

```diff
@@ -153,14 +153,13 @@ function authorIsBendyButtV1() {
 }
 
 function isRoot() {
-  return equal(seekRoot, null, {
+  return equal(seekRoot, undefined, {
```

Comment:
This is the correct way to check for this. In the old jitdb code null worked the same way as undefined, so this means the root index will continue to work and don't need to be reindexed, but we need to do this properly now, hence this change.

Reply:
@arj03 Do you think that if this were changed to use `absent()`, would it cause a reindex?

Reply:
It should not cause a reindex. What I'd like to do before we merge this is to run some queries using the new absent and check if this behaves exactly like it did before. I belief right now is that this is the same as before the change, but I need to double check. I'll let you know later this afternoon :)

Reply:
I'm thinking that maybe it would cause a new index to be built?
Previously there is `value_content_root_.index` and then there would be `value_content_root__absent.index` unless we add a way to override the indexName.

Reply:
So I can some local tests on a around 1GB test database I have migrated from db1 to db2. It is maybe ½ a year old. I have been running these 2 queries:

```
ssb.db.query(
  where(
    and(
      isRoot(),
      author('@6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ed25519')
    ),
```

and

```
ssb.db.query(
  where(
    and(
      isRoot(),
      type('post')
    ),
```
on both this branch and master I get the exact same number of messages for both queries.

I have tried replacing isRoot() with this function:

```
function isAbsent() {
  return absent(seekRoot, {
    indexType: 'value_content_root',
  })
}
```
and that also gives the exactly same number of messages.

You are correct that changing isRoot to be absent would require a reindex. I think we should leave it as is in order not to force unnecessary reindex, but I'm not super strict on that requirement if you'd rather have absent :) 

I noticed this seems a bit faster compared to the benchmarks in #277 but I think that is just the CI running on a different machine. I'll double check locally. JITDB at least is exactly the same speed locally.

Reply:
Yep, benchmarks locally are within random fluctuations of neurons hitting the earth

Reply:
Yeah, I'm with you that we don't need to change `isRoot` to use `absent()`. Anyway where we are at the moment with ssb-db2 there are several queries that would give the same results, so this is one of those cases. The refactor to `absent()` would be aesthetically neat but not worth it.